rotkehlchen/api/rest.py

```diff
@@ -3568,11 +3570,11 @@ def get_history_events(
                 group_by='event_identifier' if group_by_event_ids else None,
             )
             location = filter_query.location
-            chain_id = None
-            if location == Location.ETHEREUM:  # todo: this should prolly become a function
-                chain_id = ChainID.ETHEREUM
-            elif location == Location.OPTIMISM:
-                chain_id = ChainID.OPTIMISM
+            if location in EVM_LOCATIONS:
+                chain_id = ChainID(Location.to_chain_id(location))
+            else:
+                chain_id = None
```

Comment:
iirc we prefer this format in such places

```suggestion
            chain_id = None
            if location in EVM_LOCATIONS:
                chain_id = ChainID(Location.to_chain_id(location)) 
```

or even

```suggestion
            chain_id = ChainID(Location.to_chain_id(location)) if location in EVM_LOCATIONS else None
```